update dependencies

tomap opened the pull request:
Hi,

I updated most dependencies and change minimum node version to 6.
(we could also remove the dependency to object-assign)

Thomas

tomap committed:
update dependencies

Kakise commented on package.json:
```diff
@@ -1,30 +1,42 @@
+    "minimatch": "3.0.4",
+    "object-assign": "4.1.1",
+    "stream-to-array": "2.3.0",
+    "uncss": "0.16.2"
```
Breaks compiling: "ENONENT"

Kakise replied:
Happens with every js file. https://app.netlify.com/sites/kakise-blog/deploys/5ae4d0f9c9659262be0ba6c5

```
9:52:59 PM: Error: Could not load script: "/lib/jquery/jquery.min.js"
9:52:59 PM:     at /opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:44:23
9:52:59 PM:     at Object.check (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:75:11)
9:52:59 PM:     at ReadStream.<anonymous> (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:94:12)
9:52:59 PM:     at ReadStream.wrappedEnqueued (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:253:16)
9:52:59 PM:     at emitOne (events.js:116:13)
9:52:59 PM:     at ReadStream.emit (events.js:211:7)
9:52:59 PM:     at fs.js:1996:12
9:52:59 PM:     at FSReqWrap.oncomplete (fs.js:135:15) { Error: ENOENT: no such file or directory, open '/lib/jquery/jquery.min.js'
9:52:59 PM:   errno: -2,
9:52:59 PM:   code: 'ENOENT',
9:52:59 PM:   syscall: 'open',
9:52:59 PM:   path: '/lib/jquery/jquery.min.js' }
9:52:59 PM: Error: Could not load script: "/lib/justified-gallery/js/jquery.justifiedGallery.min.js"
9:52:59 PM:     at /opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:44:23
9:52:59 PM:     at Object.check (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:75:11)
9:52:59 PM:     at ReadStream.<anonymous> (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:94:12)
9:52:59 PM:     at ReadStream.wrappedEnqueued (/opt/build/repo/node_modules/uncss/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:253:16)
9:52:59 PM:     at emitOne (events.js:116:13)
9:52:59 PM:     at ReadStream.emit (events.js:211:7)
9:52:59 PM:     at fs.js:1996:12
9:52:59 PM:     at FSReqWrap.oncomplete (fs.js:135:15) { Error: ENOENT: no such file or directory, open '/lib/justified-gallery/js/jquery.justifiedGallery.min.js'
9:52:59 PM:   errno: -2,
9:52:59 PM:   code: 'ENOENT',
9:52:59 PM:   syscall: 'open',
9:52:59 PM:   path: '/lib/justified-gallery/js/jquery.justifiedGallery.min.js' }
```

tomap replied:
Hi, do you have the same issue without this PR?
I believe the issue you have is because uncss is executing jquery (the JS in your page) before executing uncss.
I do not have this issue because I do not have JS on my pages

Kakise replied:
Nope, I don't have this issue without this PR (I have errors coming from the js in <script></script> tags though)

Kakise replied:
This line: https://github.com/Kakise/hexo-uncss-stylus/blob/35668fdc8ba765fb4b506d308daa6ee5005708a9/index.js#L45 seems to fix the problem with both <script></script> and js files but the filter isn't executed automatically anymore. I don't know how to fix this.  It also fixes this issue: https://github.com/Kakise/Blog/issues/24 but well, you have to specify the files manually.

tomap replied:
I'l not sure what the "phases of generations" are, but semanticaly speaking, this seems like a wrong fix (playing with the CSS has not much to do with the HTML)

And what happens if you change priority without changing the filter?
so if you decrease priority (>10) ?

Kakise replied:
Same result unfortunately

Kakise reviewed (changes requested):
Please correct the bug with js files.

To reproduce, install your version of the plugin with no .uncssrc file and leave the default conf.

tomap commented:
I just did an installation of the current version of hexo-uncss and it simply does not work:
See https://github.com/tomap/Blog
(I dropped the critical css plugin which was taking forever to process files, even after the update)

When you try `hexo generate -f`, no exception, it works, but the result css for css/style.css seems really empty.
Maybe it is not compatible with other plugins like lazy loading

When you try `hexo server -o` lots of errors that do not occur without this plugin.
Maybe something to do with promises, not sure

However, I'm sure my module update has nothing to do with the error you have.

Your error is caused by another npm package:
https://github.com/Kakise/Blog/commit/23c7b60e8053686277197d67a0986b4fbc2c53f2#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R68
https://github.com/Kakise/hexo-uncss
which does not exist anymore?

I believe we can safely complete this PR.
You should open an issue for your problem, and maybe try to isolate your problem by disabling your plugins, except the one that is causing the instability when used with hexo-uncss

Kakise commented:
You're right, after extensive testing, it was another plugin's fault !

tomap committed:
drop object-assign dependency and update bluebird to latest and switch to loose version in package.json

tomap committed:
update deps

tomap committed:
update deps

tomap commented:
Up ?

tomap committed:
update dependencies + switch to eslint

tomap commented:
Up again ?

tomap committed:
update dependencies

tomap committed:
Delete package-lock.json

tomap committed:
update dependencies

tomap committed:
drop useless files from npm

tomap committed:
Update depdendencies

tomap committed:
update dependencies

tomap committed:
update dependencies

tomap committed:
update dependencies

tomap committed:
fix min version of node

tomap committed:
update dependencies

tomap committed:
update dependencies

tomap committed:
update dependencies

tomap committed:
update

tomap committed:
Fork

tomap committed:
Fix CI

tomap committed:
ignore github ci folder

tomap committed:
Fix ci

tomap committed:
update dependencies

tomap committed:
update dependencies

dependabot-preview[bot] committed:
Update eslint requirement from ^7.9.0 to ^7.10.0

Updates the requirements on [eslint](https://github.com/eslint/eslint) to permit the latest version.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.9.0...v7.10.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

tomap committed:
Merge pull request #1 from tomap/dependabot/npm_and_yarn/eslint-tw-7.10.0

Update eslint requirement from ^7.9.0 to ^7.10.0

dependabot-preview[bot] committed:
Update eslint requirement from ^7.10.0 to ^7.11.0 (#2)

Updates the requirements on [eslint](https://github.com/eslint/eslint) to permit the latest version.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.10.0...v7.11.0)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

tomap committed:
Rename repo

tomap committed:
update dependencies

tomap committed:
up

tomap committed:
up

tomap committed:
update dev dependencies

tomap committed:
update

dependabot-preview[bot] committed:
Upgrade to GitHub-native Dependabot (#24)

Co-authored-by: dependabot-preview[bot] <27856297+dependabot-preview[bot]@users.noreply.github.com>

tomap committed:
update dependencies

tomap committed:
up

tomap committed:
Fix ci

tomap committed:
Update npm-publish.yml

tomap committed:
fix

tomap committed:
Update npm-publish.yml

tomap committed:
Create codeql-analysis.yml

dependabot[bot] committed:
Update eslint-config-hexo requirement from ^4.1.0 to ^4.2.0 (#38)

Updates the requirements on [eslint-config-hexo](https://github.com/hexojs/eslint-config-hexo) to permit the latest version.
- [Release notes](https://github.com/hexojs/eslint-config-hexo/releases)
- [Commits](https://github.com/hexojs/eslint-config-hexo/compare/4.1.0...4.2.0)

---
updated-dependencies:
- dependency-name: eslint-config-hexo
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Update debug requirement from ^4.3.1 to ^4.3.2 (#33)

Updates the requirements on [debug](https://github.com/visionmedia/debug) to permit the latest version.
- [Release notes](https://github.com/visionmedia/debug/releases)
- [Commits](https://github.com/visionmedia/debug/compare/4.3.1...4.3.2)

---
updated-dependencies:
- dependency-name: debug
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Update debug requirement from ^4.3.2 to ^4.3.3 (#48)

Updates the requirements on [debug](https://github.com/debug-js/debug) to permit the latest version.
- [Release notes](https://github.com/debug-js/debug/releases)
- [Commits](https://github.com/debug-js/debug/compare/4.3.2...4.3.3)

---
updated-dependencies:
- dependency-name: debug
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump eslint from 7.32.0 to 8.3.0 (#47)

Bumps [eslint](https://github.com/eslint/eslint) from 7.32.0 to 8.3.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.32.0...v8.3.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump mocha from 8.4.0 to 9.1.3 (#44)

Bumps [mocha](https://github.com/mochajs/mocha) from 8.4.0 to 9.1.3.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v8.4.0...v9.1.3)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Node 12+

tomap committed:
update dependencies

dependabot[bot] committed:
Bump github/codeql-action from 1 to 2 (#81)

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 1 to 2.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/v1...v2)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/checkout from 2 to 3 (#80)

Bumps [actions/checkout](https://github.com/actions/checkout) from 2 to 3.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v2...v3)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/setup-node from 2 to 3 (#79)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 2 to 3.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v2...v3)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
bump build

tomap committed:
fix

dependabot[bot] committed:
Bump mocha from 9.2.2 to 10.2.0 (#83)

Bumps [mocha](https://github.com/mochajs/mocha) from 9.2.2 to 10.2.0.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/master/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v9.2.2...v10.2.0)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump minimatch from 5.1.6 to 6.1.6 (#87)

Bumps [minimatch](https://github.com/isaacs/minimatch) from 5.1.6 to 6.1.6.
- [Release notes](https://github.com/isaacs/minimatch/releases)
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v5.1.6...v6.1.6)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

dependabot[bot] committed:
Bump minimatch from 6.2.0 to 7.1.1 (#90)

Bumps [minimatch](https://github.com/isaacs/minimatch) from 6.2.0 to 7.1.1.
- [Release notes](https://github.com/isaacs/minimatch/releases)
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v6.2.0...v7.1.1)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

dependabot[bot] committed:
Bump minimatch from 7.4.6 to 9.0.0 (#93)

Bumps [minimatch](https://github.com/isaacs/minimatch) from 7.4.6 to 9.0.0.
- [Release notes](https://github.com/isaacs/minimatch/releases)
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v7.4.6...v9.0.0)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Fix

tomap committed:
bump minor

Bump node requirements to node 14

tomap committed:
bump node

tomap committed:
Update npm-publish.yml

tomap committed:
bump

dependabot[bot] committed:
Bump actions/checkout from 3 to 4 (#94)

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

dependabot[bot] committed:
Bump actions/setup-node from 3 to 4 (#95)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 3 to 4.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

tomap committed:
minor bump

dependabot[bot] committed:
Bump github/codeql-action from 2 to 3 (#96)

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 2 to 3.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/v2...v3)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump eslint from 8.57.0 to 9.1.0 (#98)

Bumps [eslint](https://github.com/eslint/eslint) from 8.57.0 to 9.1.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/main/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v8.57.0...v9.1.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

tomap committed:
Update .eslintrc

tomap committed:
Delete .eslintignore

dependabot[bot] committed:
Bump minimatch from 9.0.5 to 10.0.1 (#99)

Bumps [minimatch](https://github.com/isaacs/minimatch) from 9.0.5 to 10.0.1.
- [Changelog](https://github.com/isaacs/minimatch/blob/main/changelog.md)
- [Commits](https://github.com/isaacs/minimatch/compare/v9.0.5...v10.0.1)

---
updated-dependencies:
- dependency-name: minimatch
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

dependabot[bot] committed:
Bump mocha from 10.8.2 to 11.0.1 (#100)

Bumps [mocha](https://github.com/mochajs/mocha) from 10.8.2 to 11.0.1.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v10.8.2...v11.0.1)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update npm-publish.yml

tomap committed:
Update package.json

dependabot[bot] committed:
Bump eslint-config-hexo from 5.0.0 to 6.0.0 (#101)

Bumps [eslint-config-hexo](https://github.com/hexojs/eslint-config-hexo) from 5.0.0 to 6.0.0.
- [Release notes](https://github.com/hexojs/eslint-config-hexo/releases)
- [Commits](https://github.com/hexojs/eslint-config-hexo/compare/5.0.0...v6.0.0)

---
updated-dependencies:
- dependency-name: eslint-config-hexo
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Update package.json

dependabot[bot] committed:
Bump actions/checkout from 4 to 5 (#102)

Bumps [actions/checkout](https://github.com/actions/checkout) from 4 to 5.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/setup-node from 4 to 5 (#103)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 4 to 5.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump github/codeql-action from 3 to 4 (#104)

Bumps [github/codeql-action](https://github.com/github/codeql-action) from 3 to 4.
- [Release notes](https://github.com/github/codeql-action/releases)
- [Changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md)
- [Commits](https://github.com/github/codeql-action/compare/v3...v4)

---
updated-dependencies:
- dependency-name: github/codeql-action
  dependency-version: '4'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Bump version from 2.3.7 to 2.3.8

dependabot[bot] committed:
Bump actions/setup-node from 5 to 6 (#105)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 5 to 6.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Bump version from 2.3.8 to 2.3.9

dependabot[bot] committed:
Bump actions/checkout from 5 to 6 (#106)

Bumps [actions/checkout](https://github.com/actions/checkout) from 5 to 6.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tomap committed:
Bump version to 2.3.10
